### DISCUSSION TITLE
Nulleinspeisung mit Tasmota und opendtu

### DISCUSSION THREAD

**Discussion by Dave24282** [tbnobody/OpenDTU#2193] [General]
Hallo habe alles eingerichtet und bekomme auch Daten etc. Leider bekomme ich es noch nicht hin die nulleinspeisung zu realisieren. Kann mir jemand eventuell helfen?

Habe einen Batteriespeicher mit victron falls dies noch hilfreich ist?

![Screenshot_20240806_193229](https://github.com/user-attachments/assets/290b861a-081a-4fff-a9d8-3bab3f8c1f60)

**Comment by stefan123t** [comment]
@Dave24282 please double check the **Dynamic Power Limiter (DPL)** function in the daughter project[ OpenDTU-OnBattery](https://github.com/helgeerbe/OpenDTU-OnBattery/).

I have described a [possible PM+DPL setup](https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352) with a Shelly Pro3EM as **Power Meter (PM)** here in issue #272.

**Comment by Dave24282** [comment]
Ich suche das richtige Script dafür für die nulleinspeisung welches ich im tasmota eintragen soll

**Reply by Omega13x** [reply]
Dann musst du in den verschiedenen Tasmota Foren nachfragen. Hier bist du falsch mit deinem Anliegen. 
Entweder hier: [https://github.com/arendst/tasmota](url) oder hier: [https://ottelo.jimdofree.com/stromz%C3%A4hler-auslesen-tasmota/](url) oder hier: [https://forum.creationx.de/forum/](url) oder hier: [https://community.simon42.com/](url)
